src/pages/iou/request/step/IOURequestStepAmount.tsx

```diff
@@ -117,7 +147,57 @@ function IOURequestStepAmount({
         // If a reportID exists in the report object, it's because the user started this flow from using the + button in the composer
         // inside a report. In this case, the participants can be automatically assigned from the report and the user can skip the participants step and go straight
         // to the confirm step.
-        if (report?.reportID) {
+        if (report?.reportID && !ReportUtils.isArchivedRoom(report)) {
+            const selectedParticipants = IOU.setMoneyRequestParticipantsFromReport(transactionID, report);
+            const participants = selectedParticipants.map((participant) => {
+                const participantAccountID = participant?.accountID ?? 0;
+                return participantAccountID ? OptionsListUtils.getParticipantsOption(participant, personalDetails) : OptionsListUtils.getReportOption(participant);
+            });
+            const backendAmount = CurrencyUtils.convertToBackendAmount(Number.parseFloat(amount));
+
+            if (shouldSkipConfirmation) {
+                if (iouType === CONST.IOU.TYPE.SPLIT) {
```

Comment:
In the case we create a split bill via QAB that means the report already exists we should call splitBill function as the same [here](https://github.com/Expensify/App/blob/944751fcec525ece0da16a6a4efd80ad8ab4ff8c/src/pages/iou/request/step/IOURequestStepConfirmation.tsx#L316) which already handle this case and also navigate to the correctly report after creating.

Reply:
I'm still seeing the wrong behavior in non-QAB flows 😕

Reply:
@Gonals Could you help to detail what is wrong?

Reply:
I've tested the flow and it creates two groups:
1. Create a new group chat
2. Add a Split request in that chat
3. Create a new split from global create. Choose the members of the chat.
4. A new group chat is created, instead of posting it in the existing one